### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by paloobi**
Created cart schema & routes, and also took a stab at localStorage in the process - none of this is tested yet, it definitely needs code review.

### CODE REVIEW

File: browser/js/cart/cart.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+})
+
+app.factory('CartFactory', function(localStorageService) {
+  var CartFactory = {};
+
+  CartFactory.getCart = function (user) {
+    if (!user) {
+      return localStorageService.get('cart');
+    } else {
+      return $http.get('/api/users/' + user._id + 'cart')
```
Comment (pdpatel2) [inline browser/js/cart/cart.js]: '/api/users/' + user._id + '/cart'

File: server/db/models/cart.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Wine',
+        required: true
+      },
+      quantity: {
+        type: Number,
+        min: 1,
+        default: 1
+      }
+})
```
Comment (pdpatel2) [inline server/db/models/cart.js]: Do we want the cart to display the price of the item and the subtotal?

Comment (paloobi) [inline server/db/models/cart.js]: It should grab price from the wine itself

**Comment by paloobi** [comment]
Looks great!

**Pull request merged**